code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -53,7 +58,9 @@ Difficulty: Very Hard
 							   /datum/action/innate/megafauna_attack/aoe_attack,
 							   /datum/action/innate/megafauna_attack/shotgun,
 							   /datum/action/innate/megafauna_attack/alternating_cardinals)
-
+	/// Have we used our final attack yet?
+	var/final_availible = TRUE
```

Comment:
Should be `final_available`, though I won't make a suggestion since it's used in a few spots

Reply:
done